apps/heureka/src/hooks/useCommunication.js

```diff
@@ -4,64 +4,20 @@
  */
 
 import { useEffect } from "react"
-import { broadcast, get, watch } from "@cloudoperators/juno-communicator"
-import {
-  useUserActivityActions,
-  useAuthAppLoaded,
-  useAuthIsProcessing,
-  useAuthError,
-  useAuthLoggedIn,
-  useAuthLastAction,
-  useAuthActions,
-} from "./useAppStore"
-import { AUTH_ACTIONS } from "../lib/slices/createAuthDataSlice"
+import { get, watch } from "@cloudoperators/juno-communicator"
 
 const useCommunication = () => {
-  const { setIsActive } = useUserActivityActions()
-  const authAppLoaded = useAuthAppLoaded()
-  const authIsProcessing = useAuthIsProcessing()
-  const authError = useAuthError()
-  const authLoggedIn = useAuthLoggedIn()
-  const authLastAction = useAuthLastAction()
-  const { setData: authSetData, setAppLoaded: authSetAppLoaded } = useAuthActions()
-
-  useEffect(() => {
-    // watch for user activity updates messages
-    // with the watcher we get the user activity object when this app is loaded before the Auth app
-    const unwatch = watch(
-      "USER_ACTIVITY_UPDATE_DATA",
-      (data) => {
-        setIsActive(data?.isActive)
-      },
-      { debug: true }
-    )
-    return unwatch
-  }, [setIsActive])
-
-  // allow heureka to login/logout the user. Visible when app is not in embedded mode
   useEffect(() => {
```

Comment:
The whole topic with the authentication could be removed. It is not yet being used and if the only use case is the support_group we could use the same pattern as supernova. What do you think?

Reply:
Definitely agree, I will remove it in a separate PR.